app/features/businessContact/forms/CompanyContactDetailsForm.scala

```diff
@@ -54,4 +54,19 @@ object CompanyContactDetailsForm {
     case CompanyContactDetails(_, None, None, _) => Invalid(validationError("atLeastOneNumber"))
     case _                                       => Valid
   }
+
+  def isValidPhoneNumber: Constraint[String] = Constraint { phone: String =>
```

Comment:
the phone number is manipulated once to remove the spaces using 'removeSpaces' then again
here ->  def isValidNumber(s: String) = s.replaceAll(" ", "").matches("^[0-9]*") then again
here -> val al digitCount = phone.trim.replaceAll(" ", "").length 
we are trimming the spaces three times?